Dockerfile

```diff
@@ -1,12 +1,26 @@
-ARG node_version
-FROM node:$node_version
-
-WORKDIR /usr/app
-
-COPY package.json .
-COPY package-lock.json .
-
+# Base Build
+FROM node:10.20.1-alpine AS build-couch2pg
+RUN apk --no-cache -q add build-base libgit2-dev bash
+RUN ln -s /usr/lib/libcurl.so.4 /usr/lib/libcurl-gnutls.so.4
+WORKDIR /app
+COPY . .
```

Comment:
double check that this magically knows if the code changes on disk and re-runs the docker image compilation from this point